website/content/docs/connect/cluster-peering/index.mdx

```diff
@@ -1,32 +1,58 @@
 
-1. Create a peering token in one cluster.
-1. Use the peering token to establish peering with a second cluster.
-1. Export services between clusters.
-1. Create intentions to authorize services for peers.
+- [HTTP API reference: `/peering/` endpoint](/consul/api-docs/peering)
+- [CLI reference: `peering` command](/consul/commands/peering).
+- [Cluster peering technical specifications](/docs/consul/cluster-peering/configuration)
 
-This process establishes cluster peering between two [admin partitions](/consul/docs/enterprise/admin-partitions). Deployments without an Enterprise license can still use cluster peering because every datacenter automatically includes a `default` partition.
+**Usage information:**
 
-For detailed instructions on establishing cluster peering connections, refer to [Create and Manage Peering Connections](/consul/docs/connect/cluster-peering/create-manage-peering).
+- [Establish cluster peering connections](/docs/connect/cluster-peering/create-manage-peering)
+- [Manage cluster peering connections](/consul/docs/cluster-peering/usage/manage-connections)
+- [L7 traffic management for cluster peering connections](/consul/docs/cluster-peering/usage/peering-traffic-management)
 
-> To learn how to peer clusters and connect services across peers in AWS Elastic Kubernetes Service (EKS) environments, complete the [Consul Cluster Peering on Kubernetes tutorial](/consul/tutorials/developer-mesh/cluster-peering-aws?utm_source=docs).
+**Runtime-specific information:**
 
-### Differences between WAN federation and cluster peering
+- [Cluster peering on Kubernetes](/consul/docs/cluster-peering/usage/k8s)
 
-WAN federation and cluster peering are different ways to connect Consul deployments. WAN federation connects multiple datacenters to make them function as if they were a single cluster, while cluster peering treats each datacenter as a separate cluster. As a result, WAN federation requires a primary datacenter to maintain and replicate global states such as ACLs and configuration entries, but cluster peering does not.
+**Tutorials:**
+
+- To learn how to peer clusters and connect services across peers in AWS Elastic Kubernetes Service (EKS) environments, complete the [Consul Cluster Peering on Kubernetes tutorial](/consul/tutorials/developer-mesh/cluster-peering-aws).
+
+## Background
+
+Consul supports cluster peering connections between two [admin partitions](/consul/docs/enterprise/admin-partitions) _in different datacenters_. Deployments without an Enterprise license can still use cluster peering because every datacenter automatically includes a default partition. Meanwhile, admin partitions _in the same datacenter_ do not require cluster peering connections because you can export services between them without generating or exchanging a peering token.
+
+The following diagram describes Consul's cluster peering architecture.
+
+![Diagram of cluster peering with admin partitions](/img/cluster-peering-diagram.png)
```

Comment:
the mesh gateway icons are bigger than the other icons, the consul servers are smaller than the other icons

can we make the squares more consistent?